google/resource_dataflow_job.go

```diff
@@ -147,6 +147,11 @@ func resourceDataflowJob() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringInSlice([]string{"WORKER_IP_PUBLIC", "WORKER_IP_PRIVATE", ""}, false),
 			},
+
+			"job_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
should I add `ForceNew: true` as well?

Reply:
`ForceNew` is only applicable on input fields.  It signifies that the field is not updatable, and if the value changes a new resource needs to be recreated.  Because `job_id` is read only, it is not needed here.  Good question!